com2tcp/com2tcp.py

```diff
@@ -7,11 +7,11 @@
 import serial
 
 
-def listen_to_tcp(tcp_conn, serial_conn):
+def listen_to_tcp(tcp_conn: socket.socket, serial_conn: serial.Serial) -> None:
     while True:
         tcp_data = tcp_conn.recv(1024)
         if len(data) > 0:
-            serial_conn.writelines(tcp_data)
+            serial_conn.writelines(tcp_data)  # type: ignore - pyright doesn't understand bytes -> readable buffer
```

Comment:
Hmm I think this is pyright actually pointing out that we're misusing the pyserial API.

I think `serial_conn.writelines([tcp_data])` is probably the solution (i.e. write one line containing all the data we received) - which pyright is happy with